app/policies/learning_hour_policy.rb

```diff
@@ -4,7 +4,7 @@ def index?
   end
 
   def show?
-    @user.casa_admin? || @user.supervisor? || record.user_id == @user.id
+    record.user_id == @user.id
```

Comment:
this seems less allowing than previously - how does this fit into the change goals?

Reply:
The previous code was introduced as part of #5499 which I decided to revert completely. Since admins/supervisors view learning hours differently from volunteers, they need to have their own controller, routes and views and therefore this affect the policies.

I make use of a different policy in the code I introduced.